live-editing/configs/AccordionConfigGenerator.ts

```diff
@@ -0,0 +1,50 @@
+import {
+    IgxAccordionModule,
+    IgxIconModule,
+    IgxRadioModule,
+    IgxTimePickerModule,
+    IgxSliderModule,
+    IgxCheckboxModule
+} from 'igniteui-angular';
+import { AppModuleConfig, Config, IConfigGenerator } from 'igniteui-live-editing';
+
+export class AccordionConfigGenerator implements IConfigGenerator {
+
+    public generateConfigs(): Config[] {
+        const configs = new Array<Config>();
+
+        configs.push(new Config({
+            component: 'AccordionSample1Component',
+            appModuleConfig: new AppModuleConfig({
+                imports: ['AccordionSample1Component', 'IgxAccordionModule'],
+                ngDeclarations: ['AccordionSample1Component'],
+                ngImports: ['IgxAccordionModule']
+            }),
+            shortenComponentPathBy: '/layouts/accordion/'
+        }));
+
+        configs.push(new Config({
+            component: 'AccordionSample2Component',
+            appModuleConfig: new AppModuleConfig({
+                imports: ['AccordionSample2Component', 'IgxAccordionModule'],
+                ngDeclarations: ['AccordionSample2Component'],
+                ngImports: ['IgxAccordionModule']
+            }),
+            shortenComponentPathBy: '/layouts/accordion/'
+        }));
+
+        configs.push(new Config({
+            component: 'AccordionSample3Component',
+            appModuleConfig: new AppModuleConfig({
+                imports: ['IgxCheckboxModule', 'IgxSliderModule',
+                    'IgxRadioModule', 'IgxIconModule', 'IgxTimePickerModule', 'AccordionSample3Component', 'IgxAccordionModule'],
+                ngDeclarations: ['AccordionSample3Component'],
+                ngImports: ['IgxCheckboxModule', 'IgxSliderModule',
+                    'IgxRadioModule', 'IgxIconModule', 'IgxTimePickerModule', 'IgxAccordionModule']
```

Comment:
I think the `IgxIconModule` is redundant here - `IgxTimePickerModule` export `IgxInputGroupModule`, which in turn export `IgxIconModule`